src/test/java/plugins/AntPluginTest.java

```diff
@@ -21,6 +21,9 @@
     public static final String INSTALL_VERSION_1_8 = "1.8.4";
     public static final String INSTALL_NAME_1_8 = "ant_" + INSTALL_VERSION_1_8;
 
+    public static final String INSTALL_VERSION_1_10 = "1.10.5";
+    public static final String INSTALL_NAME_1_10 = "ant_" + INSTALL_VERSION_1_10;
```

Comment:
Please inline these since they are used only once. There is no reason to have them public.

Reply:
Changed them to private. I like to keep "magic" strings as constants.